sites/example-project/src/components/viz/FunnelChart.svelte

```diff
@@ -122,7 +122,8 @@
         },
         label: {
             show: true,
-            position: labelPosition
+            position: labelPosition,
+            formatter:  '{c} \n {d}%'
```

Comment:
> whitespace below the funnel, but it's a bit tight to the legend at the top - any idea what's causing that?

Yeah just had a look, its coming from the `bottom` property of [series funnel](https://echarts.apache.org/en/option.html#series-funnel.bottom) config. I've updated it to be slightly shorter to reduce whitespace.

> couple more variants to the [example-project page](https://github.com/evidence-dev/evidence/blob/main/sites/example-project/src/pages/charts/funnel-chart.md)

Cool, I've added these new variants: Funnel Sort: Ascending and Funnel Align: Right and Left

> updated images / examples in the docs 

Sure, I've added/updated images in the docs, but do let me know if anything needs to be changed.

> Number Formatting and Percentages 
Updated, however, the percentages in the same line in the `funnelAlign = right` is a bit of an issue since it does cut off the bracket in the end. That does not happen if the percent is on a different line.